README.md

```diff
@@ -11,7 +11,7 @@ This sdk is designed to help build client side and server side content managed a
 
 ## Features
 
-- Fetch content and slots using the [Content Delivery Service](https://docs.amplience.net/integration/deliveryapi.html#the-content-delivery-api)
+- Fetch content and slots using the [Content Delivery Service v1](https://docs.amplience.net/integration/deliveryapi.html#the-content-delivery-api) and/or [Content Delivery Service v2](https://docs.amplience.net/development/contentdelivery/readme.htm)
```

Comment:
I'd be interested in other views, but... the naming for this API is "Content Delivery 2". It used to be "Content Delivery v2" but the 'v' has been dropped. Except, to distinguish it from the other one, we now call the old one 'Content Delivery v1' even though that's not its real name. Such confuse.

Maybe the right answer is to have _Content Delivery Service v1_ and _Content Delivery 2_. Strange.

Reply:
https://semver.org/ ?

Reply:
CD2 is a separate API to CDv1 rather than a replacement.

Reply:
Well thats confusing, as version 2 assumes that it will replace version 1 :)